src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -291,17 +292,21 @@ class TooltipHTMLContent {
         (el as any).domBelongToZr = true;
         this.el = el;
         const zr = this._zr = api.getZr();
-        const appendToBody = this._appendToBody = opt && opt.appendToBody;
 
-        makeStyleCoord(this._styleCoord, zr, appendToBody, api.getWidth() / 2, api.getHeight() / 2);
+        const appendTo = opt.appendTo;
+        const container: HTMLElement = (
+            isString(appendTo)
+                ? document.querySelector(appendTo)
+                : isDom(appendTo)
+                    ? appendTo
+                    : isFunction(appendTo) && appendTo(api.getDom())
+        );
```

Comment:
I think the fallback is required. The default container is just `api.getDom()`. It will be undefined if no fallback here.

Reply:
You are right. I wrote `(container || api.getDom()).appendChild(el);` in the nextline but haven commit yet.